sqflite/lib/src/connection.dart

```diff
@@ -0,0 +1,194 @@
+import 'dart:async';
+
+import 'package:jaguar_query/jaguar_query.dart';
+import 'package:jaguar_query_sqflite/composer.dart';
+import 'package:sqflite/sqflite.dart' as sqf;
+
+class SqfConn extends Connection<sqf.Database> {
```

Comment:
rename `SqfliteConnection` please to be more consistent with other names

Reply:
In the postgres the named the same class as PgConn. I was following the same code style...

Reply:
I guess you shouldn't, all that short naming stuff is super confusing, maintaining and onboarding new contributors with this kind of stuff is complicated imho.